lib/features/digital_guide_view/tabs/adapted_toilets/presentation/adapted_toilet_detail_view.dart

```diff
@@ -0,0 +1,140 @@
+import "package:auto_route/auto_route.dart";
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../../../../config/ui_config.dart";
+import "../../../../../theme/app_theme.dart";
+import "../../../../../utils/context_extensions.dart";
+import "../../../../../widgets/detail_views/detail_view_app_bar.dart";
+import "../../../../../widgets/my_cached_image.dart";
+import "../../../../../widgets/my_point_widget.dart";
+import "../../../presentation/widgets/digital_guide_nav_link.dart";
+import "../data/models/adapted_toilet.dart";
+
+@RoutePage()
+class AdaptedToiletDetailView extends ConsumerWidget {
+  const AdaptedToiletDetailView({
+    required this.adaptedToilet,
+  });
+
+  final AdaptedToilet adaptedToilet;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Scaffold(
+      appBar: DetailViewAppBar(
+        actions: const [],
+      ),
+      body: Padding(
+        padding: const EdgeInsets.symmetric(
+          horizontal: DigitalGuideConfig.heightBig,
+        ),
+        child: CustomScrollView(
+          slivers: [
+            SliverList(
+              delegate: SliverChildListDelegate([
+                Text(
+                  adaptedToilet.getDescription(context),
+                  style: context.textTheme.title.copyWith(fontSize: 18),
+                ),
+                const SizedBox(
+                  height: DigitalGuideConfig.heightMedium,
+                ),
+                if (adaptedToilet.translations.plTranslation.comment.isNotEmpty)
+                  MyPointWidget(
+                    text: adaptedToilet.translations.plTranslation.comment,
+                  ),
+                MyPointWidget(
+                  text: adaptedToilet.translations.plTranslation.location,
+                ),
+                if (adaptedToilet
+                    .translations.plTranslation.numberOfCabins.isNotEmpty)
+                  MyPointWidget(
+                    text:
+                        adaptedToilet.translations.plTranslation.numberOfCabins,
+                  ),
+                MyPointWidget(
+                  text: adaptedToilet
+                      .translations.plTranslation.toiletDescription,
+                ),
+                if (!adaptedToilet.isAccessAccessibleForPwd)
+                  MyPointWidget(
+                    text: adaptedToilet.translations.plTranslation
+                        .isAccessAccessibleForPwdComment,
+                  ),
+                if (adaptedToilet.isNeedAuthorization != "Unknown")
+                  MyPointWidget(
+                    text: adaptedToilet.isNeedAuthorization == "True"
```

Comment:
Shoudlnt we here define some kind of enum? Since it appears that `isNeedAuthorization` could return predictable strings, eg. always Unknown or True or else if ok